### PR TITLE
Improvements to Wiki focused on contribution guidence

### LINKED ISSUE #331: Improve CONTRIBUTING documentation
### User Story
As a developer new to this project
I'd like to understand how to contribute
So that I can help this great cause

### Acceptance Criteria
- [ ] Contribution guidance should be easily accessible from CONTRIBUTING and the mkdocs-based wiki, ideally displaying the same content in both.

### Notes
For ideas on what make great contribution guidance, check out this article: https://embeddedartistry.com/blog/2017/12/11/get-others-involved-in-your-project-with-a-contributing-guide/

### PR DESCRIPTION
**Pull request by blakenan-bellese**
# Description

This PR updates the documentation wiki to provide better guidance on how to contribute and consolidate a few pages that were slightly redundant.  Take a look at the "How To Contribute" page for the majority of changes.

## Related PRs
resolves #331 

## To Review
Recommend running `mkdocs serve` to serve up a local instance of the wiki for review.

### CODE REVIEW

File: docs/HowToContribute.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+- `P1`, `P2`, `P3` each describe priority with P1 being the most urgent
+- `bug` is a defect that needs to be fixed
+- `enhancement` is a new valuable new feature 
+
+## Technology and Code
+
+This project will make exclusive use of open-source software, packages, and contributions. The application is built with the following
+technologies:
+
+- Javascript (ES6)
```
Comment (carorm) [inline docs/HowToContribute.md:50]: Link for this?

**Review by carorm — CHANGES_REQUESTED** [review]
@blakenan-bellese just minor things/opinions. You can refute if you wish but let me know and I can discuss.

File: docs/HowToContribute.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+# Welcome
+
+The CodeForBaltimore team is thrilled that you might be interested in helping us help others. You do not need to be an expert, just willing to help.  This page will give you some background on this project with suggestion and advice on how to contribute.
+
+## What is Bmore Responsive?
+
+A simple, flexible API to support emergency response coordination.  Sample use cases include:
+
+- keeping track of local nursing home status and needs during a global pandemic
```
[the remark this answers is not in the document]
Comment (blakenan-bellese) [reply docs/HowToContribute.md:9]: done

File: docs/HowToContribute.md (hunk 1 above)
Comment (blakenan-bellese) [reply docs/HowToContribute.md:50]: done

**Pull request merged**